src/hotspot/share/services/diagnosticCommand.hpp

```diff
@@ -981,7 +981,7 @@ class CompilationMemoryStatisticDCmd: public DCmdWithParser {
   virtual void execute(DCmdSource source, TRAPS);
 };
 
-#ifdef LINUX
+#if defined(LINUX) || defined(_WIN64)
```

Comment:
We also need to update the description for the diagnostic commands. Currently, both `System.dump_map` and `System.map` state they're enabled for linux only.

Reply:
A good catch - I'd already found this while preparing a macOS implementation.
Fixed.